scripts/mac_refresh.py

```
@@ -0,0 +1,92 @@
+
+def get_mac_slack_key():
+    cmd = ["security", "find-generic-password", "-s", "Slack Safe Storage", "-w"]
+    try:
+        return subprocess.check_output(cmd, stderr=subprocess.DEVNULL).strip()
+    except:
+        import subprocess
+        try: return subprocess.check_output(cmd, stderr=subprocess.DEVNULL).strip()
+        except: return None
+
+def decrypt_payload(key, ciphertext):
+    if not ciphertext or not key: return None
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)